# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/test/webgl-conf/mochitest-errata.ini
# Commit: e1b0906509ef
# Full Hash: e1b0906509efa6433978b9f980ec639b7eafbc6d
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-01-09 03:40:18
# Regressor Bug: 1477756
# File Overlap Count: 1
# Description:
#   Bug 1477756 - Fix non-webgl CI tests. r=handyman
#   
#   * Revert some partial webgl+oop+vr code.
#   * More missing FuncScope.
#   * Fix compile errors.
# ==============================================================================

diff -r 7e2a2b1b416f -r e1b0906509ef dom/canvas/test/webgl-conf/mochitest-errata.ini
--- a/dom/canvas/test/webgl-conf/mochitest-errata.ini	Wed Jan 08 22:19:23 2020 +0000
+++ b/dom/canvas/test/webgl-conf/mochitest-errata.ini	Wed Jan 08 22:19:26 2020 +0000
@@ -803,7 +803,7 @@
 fail-if = (os == 'linux')
 [generated/test_conformance__rendering__blending.html]
 # Expected [1000,1,1,1], was [1,1,1,1]
-fail-if = (os == 'android') || (os == 'linux')
+fail-if = (os == 'android')
 [generated/test_2_conformance__rendering__blending.html]
 fail-if = (os == 'android')
 [generated/test_2_conformance2__glsl3__valid-invariant.html]